### PR TITLE
Package manager guides

### PR DESCRIPTION
**Pull request by wildmanonline**
- [New] Overview of Package Management in Linux
- [New] Using APT to Manage Packages in Debian and Ubuntu
- [New] Using Portage to Manage Packages in Gentoo
- [New] Using Pacman to Manage Packages in Arch
- [New] Managing Packages in Slackware
- [Updated] Using YUM to Manage Packages in CentOS/RHEL 7 and Earlier
- [Updated] Using DNF to Manage Packages in CentOS/RHEL 8 and Fedora
- [Deprecated] Linux Package Management

Summary of changes:
- Created new section called "Linux Package Management"
- Moved the Linux Package Management guide into this section and deprecated it
- Created a new Linux Package Management guide and individual guides for each package manager (some are based on content in the original guide, some are completely new)

If this PR is merged, PR #4431 can be closed (Northcutt's proposed revisions to the original guide).

### CODE REVIEW

File: docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    Searches the package database for a package name and version number.
+
+-   `pacman --sync [package]` or `pacman -S`
+
+    Installs new packages, downloads new content for the database and/or upgrades the system, depending on the options and the named package or packages.
+
+-   `pacman --remove [package]` or `pacman -R`
+
+    Removes the named package or packages. Note that the terse flags are all uppercase and case-sensitive. These terse flags are often combined with additional flags for additional functionality. Here are some examples with brief descriptions:
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md:48]: I feel like we should break the list here, have a standalone sentence about the terse flags, then start a new list with the other options.

File: docs/guides/tools-reference/linux-package-management/portage-package-manager/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+This guide addresses common package management tasks and functions using the `emerge` front end for the portage system. We encourage you to install the "gentoolkit" to provide additional package management tools, such as `equery`. Install this package with the following command:
+
+    sudo emerge app-portage/gentoolkit
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running Gentoo.** Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/portage-package-manager/index.md:32]: Getting Started **guide**

File: docs/guides/tools-reference/linux-package-management/slackware-package-management/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Packages can also be built using SlackBuilds, shell scripts that compile source or repackage binary distribution packages for easy installation and removal on Slackware. These scripts have been adopted as a community standard by such sites as [SlackBuilds.org](https://slackbuilds.org/), which provides many common third-party packages not available in Slackware proper.
+
+Slackware includes `pkgtool` for local package management and `slackpkg` for remote installation of packages from official mirrors. For less interactive tasks, there are `installpkg`, `upgradepkg`, and `removepkg`.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running Slackware.** Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/slackware-package-management/index.md:30]: The Getting Started **guide**

File: docs/guides/tools-reference/linux-package-management/yum-package-manager/index.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+This guide aims to familiarize you with the YUM commands you are most likely to encounter. By the end, you should feel comfortable navigating YUM in all but its more advanced features. And for those, you can find some helpful resources at the end of this guide.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running on CentOS/RHEL 7, Fedora 21, or earlier versions of either distribution.** Other Linux distributions that employ the YUM package manager can also be used. Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/yum-package-manager/index.md:34]: Getting Started **guide**

File: docs/guides/tools-reference/linux-package-management/apt-package-manager/index.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+This guide aims to walk you through using APT and its command-line tools to perform common functions related to package management. The commands and examples used throughout this guide default to using the `apt` command. Many of the commands interchangeable with either `apt-get` or `apt-cache`, though there may be breaking differences.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running on Debian or Ubuntu.** Other Linux distributions that employ the APT package manager can also be used. Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/apt-package-manager/index.md:30]: Getting Started **guide**

File: docs/guides/tools-reference/linux-package-management/dnf-package-manager/index.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+
+While the YUM package manager is no longer used on these distributions, the `yum` command still works in many cases. Most distributions link the `yum` command to the DNF software and, since DNF maintains compatibility with much of YUM's CLI, most commands still function as intended. This is why some documentation for these distributions still reference the `yum` command to install or update software.
+
+This guide aims to familiarize you with the DNF commands you are most likely to encounter while working with your CentOS 8 or Fedora server. By the end, you should feel comfortable navigating all but the more advanced features DNF offers. You can also find a few helpful resources at the end of this guide.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running on CentOS/RHEL 8, AlmaLinux 8, Rocky Linux 8, Fedora 22, or later versions of these distributions.** Other Linux distributions that employ the DNF package manager can also be used. Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/dnf-package-manager/index.md:44]: Getting Started **guide**

File: docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+enable_h1: true
+---
+
+Arch Linux uses binary packages in a `.tar.xz` format, and also provides a "ports" build system that facilitates building packages. Arch Linux runs on a *rolling release* schedule, which means packages are added to the main repository when they (and their dependencies) are ready for production. This means that there aren't release versions of Arch, as all systems, once upgraded, are equivalent. Therefore, administrators of Arch Linux must consider the output of `pacman` carefully before agreeing to upgrade or update any packages.
+
+## Before You Begin
+
+Before running the commands within this guide, you will need:
+
+1. **A system running Arch Linux** or an Arch-based distribution like Manjaro. Review the [Getting Started](/docs/getting-started/) if you do not yet have a compatible system.
```
Comment (andystevensname) [inline docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md:26]: Getting Started **guide**

**Review by andystevensname — CHANGES_REQUESTED** [review]

File: docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md (hunk 1 above)
Comment (wildmanonline) [reply docs/guides/tools-reference/linux-package-management/pacman-package-manager/index.md]: Let me know if my revisions were what you intended. Essentially put a paragraph break after that first sentence and modified the rest as follows:

> Each of the above single character flags (`-Q`, `-S`, and `-R`) are uppercase. They are often combined with additional flags (also case-sensitive) to alter the functionality of the command. Here are some examples with brief descriptions:

**Review by andystevensname — APPROVED** [review]

**Pull request merged**